frontend/src/constants/API.ts

```diff
@@ -70,6 +70,7 @@ export interface IGeoSearchParams {
   floorCount?: number;
   rentableArea?: number;
   propertyType?: string;
+  tileKey?: string;
```

Comment:
What's this for?

Reply:
This does not get sent to the api, its a local field used to track the tile used to generate the bbox. This does not affect the outcome of the api request